app/controllers/application_controller.rb

```diff
@@ -2,8 +2,7 @@
 
 class ApplicationController < ActionController::Base
   include Clearance::Controller
-  before_action :require_login, :require_otp_setup
-  before_action :set_organization
+  before_action :require_login, :require_otp_setup, :user_permitted?, :set_organization
```

Comment:
I would just reorder to
`  before_action :require_login, :require_otp_setup, :set_organization, :user_permitted?`